app/views/loans/_multi_loan_form.erb

```diff
@@ -1,6 +1,6 @@
 <% if true %>
   <%= form_tag(loan_multi_path, method: "post") do %>
-    <% Book.where(selected: true).each do |b| %>
+    <% Book.where(id: session[:selected_books]).each do |b| %>
```

Comment:
I've never seen this syntax before- it's wacky! Totally counterintuitive, but it works and is so much simpler than the alternative.